Backoffice/NavigationPanel/Strategies/AbstractNavigationPanelStrategy.php

```diff
@@ -92,4 +109,22 @@ protected function render($view, array $parameters = array())
     {
         return $this->templating->render($view, $parameters);
     }
+
+    /**
+     * @param array               $datatableParameter
+     * @param TranslatorInterface $translator
+     *
+     * @return array
+     */
+    protected function preFormatDatatableParameter(array $datatableParameter, TranslatorInterface $translator)
+    {
+        foreach ($datatableParameter as $name => &$parameters) {
+            $parameters['name'] = $name;
+            if (array_key_exists('title', $parameters) && $translator) {
```

Comment:
why test $translator exists ? you check already in signature of method if $translator is an instance of TranslatorInterface